Topics/Tech_Stacks/Shadcn.md

```diff
@@ -0,0 +1,144 @@
+# Using shadcn/ui
+
+## Table of Contents
+
+#### [Introduction](#introduction-1)
+
+#### [Installation](#installation-1)
+
+#### [Creating and Customizing Components](#creating-and-customizing-components-1)
+
+## Introduction
+
+### What is shadcn/ui? 🤔
+
+[Shadcn/ui](https://ui.shadcn.com/) gives you the ability to use re-usable components that are built using [Radix UI](https://www.radix-ui.com/) and [Tailwind CSS](https://tailwindcss.com/). You are able to choose from their collection of pre-built components, add the code into your project, and then customize them using cva variants.
+
+### What to Expect 📄
+
+This document will go over how to install and integrate shadcn/ui into your project. Then, it will go through an example of building a `Button` component into your project and talk through the details in each step.
+
+## Installation
+
+You can use shadcn/ui components in any framework that supports React. Typescript is recommended for your projects but Javascript components are also available by making changes to your configurations.
+
+Follow the link below depending on the framework you are using for your project:
+
+- [Next.js](https://ui.shadcn.com/docs/installation/next)
+- [Vite](https://ui.shadcn.com/docs/installation/vite)
+- [Remix](https://ui.shadcn.com/docs/installation/remix)
+- [Gatsby](https://ui.shadcn.com/docs/installation/gatsby)
+- [Astro](https://ui.shadcn.com/docs/installation/astro)
+- [Laravel](https://ui.shadcn.com/docs/installation/laravel)
+
+If you want to manually install shadcn/ui, follow this [tutorial](https://ui.shadcn.com/docs/installation/manual).
+
+## Creating and Customizing Components
+
+<!-- This [document](https://ui.shadcn.com/docs/components/button) shows the steps you need to take in order to install and use a `Button` component in your project.  -->
+
+Follow these steps to install, use, and customize a `Button` component in your own project:
```

Comment:
Do you think readers will benefit from seeing an additional example, for a more complex component? Possible additions can be Alert Dialog or Card

Reply:
Although additional examples may be helpful, I think I agree more with Jasper's point. As an introduction page, I think it's better to focus on one clear, simple example that users can follow. To add more resources, I can also link some additional examples so students can learn more about other components on their own!

Reply:
I see, it makes sense not to give too much information so the reader isn't overwhelmed.